app/javascript/src/components/GameDetails.vue

```diff
@@ -47,7 +47,7 @@
       ></Rondel>
     </div>
     <div
-      v-if="game.importing && (profile.username === controllingPlayerName || game.soloMode)"
+      v-if="game.importing && (profile.username === controllingPlayerName || (game.soloMode && profile.username in game.players))"
```

Comment:
Can this be written as `game.players.includes(profile.username)`? I think that's more idiomatic since the `in` word is generally used for iterating through the collection, if I understand correctly.